volumegroup/README.md

```diff
@@ -205,6 +205,10 @@ message ModifyVolumeGroupMembershipRequest {
   // This field is OPTIONAL. Refer to the `Secrets Requirements`
   // section on how to use this field.
   map<string, string> secrets = 3 [(csi.v1.csi_secret) = true];
+
+  // parameters passed to the plugin to modify the volume group.
```

Comment:
The commit message mentions that the parameters can be used to modify the volumes in the group. Here the parameters are for the group itself. Which one is it, or both? Consider adding a little description for it above the protobuf message in the parargaph.

Reply:
update the commit and comment to match the same, it's up to the plugin what it wants to do with it as its opaque parameters from the spec point.